src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -166,6 +166,20 @@ case class AttributeValueList(val list: Seq[AttributeValue]) extends AttributeVa
 case class AttributeReferenceList(val list: Seq[AttributeReferenceSingle]) extends AttributeReference // non-recursive
 case class AttributeReferenceSingle(val entityType: String, val entityName: String) extends AttributeReference
 
+@ApiModel(value = "JobDescription")
+case class JobDescription(
+  @(ApiModelProperty@field)(required = true, value = "The method to execute.")
+  methodName: MethodConfigurationName,
+  @(ApiModelProperty@field)(required = true, value = "The entity that supplies parameters to the method.")
+  entityName: String)
```

Comment:
The error is pretty specific:
RequestComplete(StatusCodes.NotFound,
"No entity of type "+entType+" named "+entName+" in workspace"+wsNamespace+"."+wsName)

Would you like me to include the redundant info in the request, or alter the error message?